docs/swarm/swarm-mode.md

```diff
@@ -0,0 +1,155 @@
+<!--[metadata]>
++++
+title = "Run Docker Engine in swarm mode"
+description = "Run Docker Engine in swarm mode"
+keywords = ["guide, swarm mode, node"]
+advisory = "rc"
+[menu.main]
+identifier="initialize-swarm-guide"
+parent="engine_swarm"
+weight=12
++++
+<![end-metadata]-->
+
+# Run Docker Engine in swarm mode
+
+When you first install and start working with Docker Engine (Engine), swarm mode
```

Comment:
This is convention that writers normally use so they can say Engine later. It's fine. I'll remove.